clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -26,12 +26,13 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.HashSet;
+import java.util.HashMap;
```

Comment:
Super nitpicky -- I think the `import` list is alphabetical, and this is out of order now.